editor/svg-editor.js

```diff
@@ -2991,7 +2992,7 @@ TODOS
 					return false;
 				}
 
-				if (attr !== 'id') {
+				if (attr !== 'id' && attr!=='class') {
```

Comment:
nit: can you please space things out here like the code around it:

  attr !== 'class'